src/actors/other_shell/update.inc.c

```diff
@@ -0,0 +1,462 @@
+#include <actors.h>
+#include <waypoints.h>
+#include <defines.h>
```

Comment:
Other shell directory name will make it difficult for people to find this shell

blue_red_shell
or
blue_and_red_shells